OMEdit/OMEdit/OMEditGUI/Modeling/ModelWidgetContainer.cpp

```diff
@@ -3067,12 +3113,7 @@ void GraphicsView::keyReleaseEvent(QKeyEvent *event)
 void GraphicsView::contextMenuEvent(QContextMenuEvent *event)
 {
   /* If we are creating the connection OR creating any shape then don't show context menu */
-  if (isCreatingConnection() ||
```

Comment:
Note. before we actually had a bug here I think. We never checked if we were creating a bitmap